pudb/run.py

```diff
@@ -1,9 +1,15 @@
-COMMAND = {"zsh": "_command_names -e"}
+COMMAND = {"zsh": "_pre_run"}
 PREAMBLE = {
     "zsh": """\
 _script_args() {
   _arguments -S -s '(-)1:script_args:_files -g "*.py"' '*: :_files'
 }
+
+_pre_run() {
+  # https://github.com/zsh-users/zsh/blob/master/Completion/Unix/Command/_sudo#L47-L51
```

Comment:
`master` could change. Please make this a versioned reference.

Reply:
Fixed. :smile: